workflow-to-js-transformer/src/test/java/com/github/cafdataprocessing/workflow/transform/WorkflowJavaScriptExecutionTest.java

```diff
@@ -695,9 +666,7 @@ public void onErrorTest() throws WorkerException, ScriptException, WorkflowTrans
                 .addFieldValue("test", "string_value")
                 .addFieldValue("CAF_ACTION_TO_EXECUTE", familyHashingActionId)
                 .documentBuilder()
-                .withCustomData()
-                .add(POST_PROCESSING_NAME, postProcessingScriptRef)
-                .documentBuilder().build();
+                .build();
 
         // TODO change this to ErrorEventObject class from worker-document once those changes aer ready and the xslt has been
         // TODO updated to use that version of worker-document
```

Comment:
I'll update this at some point this week. It doesn't need to happen as part of this pull request.